index.js

```diff
@@ -82,49 +109,80 @@ var TRXReporter = function (baseReporterDecorator, config, emitter, logger, help
         counters = resultSummary.ele('Counters');
         testDefinitions = testRun.ele('TestDefinitions');
 
-        testListIdNotInAList = "8c84fa94-04c1-424b-9868-57a2d4851a1d";
+        testListIdNotInAList = '8c84fa94-04c1-424b-9868-57a2d4851a1d';
         var testLists = testRun.ele('TestLists');
 
-        testLists.ele('TestList')
+        testLists
+            .ele('TestList')
             .att('name', 'Results Not in a List')
             .att('id', testListIdNotInAList);
 
         // seems to be VS is expecting that exact id
-        testLists.ele('TestList')
+        testLists
+            .ele('TestList')
             .att('name', 'All Loaded Results')
-            .att('id', "19431567-8539-422a-85d7-44ee4e166bda");
+            .att('id', '19431567-8539-422a-85d7-44ee4e166bda');
 
         testEntries = testRun.ele('TestEntries');
         results = testRun.ele('Results');
     };
 
-    this.onBrowserStart = function(browser) {
-    };
-
-    this.onBrowserComplete = function (browser) {
-        var result = browser.lastResult;
+    this.onBrowserStart = function(browser) {};
 
-        var passed = result.failed <= 0 && !result.error;
+    this.onBrowserComplete = function(browser) {
+        var total = 0;
+        var executed = 0;
+        var successful = 0;
+        var error = 0;
+        var failed = 0;
+        if (browser.lastResult._realLastResults) {
+            console.log(
+                'Merging result(s) from parallel test run(s) into the trx report.'
+            );
+            try {
+                var realLastResult = browser.lastResult._realLastResults;
+                for (var i in realLastResult) {
```

Comment:
I think we never thought about supporting multiple browsers, I see the problem now. I guess onBrowserComplete is called for each browser. I wouldn't use _realLastResults, as this is some internal and doesn't seem to be part of the official contract, no other reporter is using that neither. In that case we should rather collect all lastResults and set the counters in the onRunComplete functions.